packages/sdk/src/contracts/operatorContractUtils.ts

```diff
@@ -188,22 +153,16 @@ export function getTestTokenContract(): DATATokenContract {
     return new Contract(TEST_CHAIN_CONFIG.contracts.DATA, DATATokenArtifact) as unknown as DATATokenContract
 }
 
-export const getTestAdminWallet = (adminKey?: string, provider?: Provider): Wallet => {
-    return new Wallet(adminKey ?? TEST_CHAIN_CONFIG.adminPrivateKey).connect(provider ?? getProvider())
-}
-
-interface GenerateWalletWithGasAndTokensOpts {
-    chainConfig?: { contracts: { DATA: string } }
+export const getTestAdminWallet = (): Wallet => {
+    return new Wallet(TEST_CHAIN_CONFIG.adminPrivateKey).connect(getTestProvider())
 }
 
-export async function generateWalletWithGasAndTokens(opts?: GenerateWalletWithGasAndTokensOpts): Promise<Wallet & SignerWithProvider> {
-    const provider = getProvider()
+export async function createTestWallet(): Promise<Wallet & SignerWithProvider> {
```

Comment:
We don't always want to spend time waiting for a wallet to be generated that contains tokens, therefore I like the old name since it indicates _what kind of wallet_ we are generating. Sometimes in tests we have wallets that don't need to tokens or gas (just generate random private key), need only gas (use KeyServer) or need both (use this function). I think the name should indicate that we are now generating a certain kind of wallet.

Reply:
Maybe it would make sense to modify `dev-chain-fast `so that they KeyServer keys have also some tokens. Then this utility function would be obsolete. https://linear.app/streamr/issue/ETH-800/dev-chain-fast-mint-data-tokens-for-pre-baked-accounts